tests/waveforms/test_arbitrary.py

```diff
@@ -1,5 +1,3 @@
-import re
```

Comment:
Surprising an unused import didn't ring any bell

Reply:
Yes, because `pre-commit` checks only files in the commit. I now run `ruff` in the whole qililab codebase